src/Dockerfile

```diff
@@ -70,8 +70,17 @@ RUN {{ spec.environment_setup }}
   {% endif %}
 {% endif %}
     INSTALL_PKGS="rsync tar gettext bind-utils nss_wrapper {{ spec.pkgs }}" && \
-{% if spec.version in ["12"] %}
-    INSTALL_PKGS="$INSTALL_PKGS rh-postgresql{{ spec.short }}-pgaudit" && \
+{% if spec.version in ["12", "13"] %}
+    {% if config.os.id == 'rhel' %}
+        {% if spec.el_version == '7' %}
+            INSTALL_PKGS="$INSTALL_PKGS rh-postgresql{{ spec.short }}-pgaudit" && \
```

Comment:
Please keep the indents inline with what is expected in the resulting Dockerfile, so that we do not get changes like this in downstream repositories for no real reason:

https://gist.github.com/rhscl-automation/a9e65f854434cca5a51bbd04fffee287#file-postgresql-container-34-diff-L45